src/components/add-meal-form/add-meal-form.jsx

```diff
@@ -0,0 +1,87 @@
+import { useState } from "react";
+import { Link } from "react-router-dom";
+import "./add-meal-form.css";
+import AddMealItem from "../addMealItem/addMealItem";
+
+function AddMealForm({ onMealSubmit }) {
+  const todayDate = new Date().toJSON().slice(0, 10);
+  const todayTime = new Date().toJSON().slice(11, 16);
+  const [mealName, setMealName] = useState("");
+  const [mealDate, setMealDate] = useState(todayDate);
+  const [mealTime, setMealTime] = useState(todayTime);
+  const [mealItem, setMealItem] = useState([]);
```

Comment:
Adiciona esse comentário aqui, depois vou te mostrar um jeito melhor
```suggestion
  // TODO: Refatorar os estados para centralizar em um objeto só
  const [mealName, setMealName] = useState("");
  const [mealDate, setMealDate] = useState(todayDate);
  const [mealTime, setMealTime] = useState(todayTime);
  const [mealItem, setMealItem] = useState([]);
```